iOS/Objective C/IronSourceDemoApp/NativeAdView.m

```diff
@@ -0,0 +1,177 @@
+//
+//  NativeAdView.m
+//  IronSourceDemoApp
+//
+//  Created by Luga Wang on 9/15/23.
+//  Copyright © 2023 supersonic. All rights reserved.
+//
+
+#import "NativeAdView.h"
+
+@implementation NativeAdView
+
+@dynamic adAppIcon;
+@dynamic adTitleView;
+@dynamic adAdvertiserView;
+@dynamic adBodyView;
+@dynamic adMediaView;
+@dynamic adCallToActionView;
+
+- (instancetype)init {
+    UINib *nib = [UINib nibWithNibName:@"LevelPlayNativeAdView" bundle:[NSBundle mainBundle]];
+    NSArray *nibContents = [nib instantiateWithOwner:nil options:nil];
+    NativeAdView *nativeAdView = (NativeAdView *)[nibContents firstObject];
+    
+    if (!nativeAdView) {
```

Comment:
why do we need this if? is there a case where (NativeAdView *)[nibContents firstObject]; wont return a value?

Reply:
solved, please review